plugin/info.vim

```diff
@@ -573,7 +585,7 @@ function! s:encodeCommand(ref, kwargs)
 		let l:cmd .= ' --file '.shellescape(a:ref['File'])
 	endif
 	if has_key(a:ref, 'Node')
-		let l:cmd .= ' --node '.shellescape(a:ref['Node'])
+		let l:cmd .= ' '.shellescape(a:ref['Node'])
```

Comment:
Why did you remove the `--node` part? Without it `GotoNode` does not work for me. Try `:GotoNode A Programmable Completion Example` in the Bash manual; with `--node` it works fine, but without nothing happens.